CMakeLists.txt

```diff
@@ -2,13 +2,20 @@
 # CMake Project for REGRADER Plugin #
 #####################################
 
+# uncomment to build as VST2.4 instead of VST3.0 (provides wider DAW compatibility)
+#set(SMTG_CREATE_VST2_VERSION "Use VST2" ON)
+
+# uncomment to build Audio Unit (requires macOS and Xcode)
+#option(JAMBA_ENABLE_AUDIO_UNIT "Enable Audio Unit" ON)
+
```

Comment:
Moving user overrides to top of file (prefer to use command line arguments though)